components/CommentCount.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+
+class CommentCount extends React.Component {
+  formatString (count) {
+    if (count == undefined) { count = 0 }
```

Comment:
Upvoting and comment count feel separate to me.

We should always be able to count the comments, no comments = 0?